pkg/apis/submariner.io/v1/types.go

```diff
@@ -81,6 +81,17 @@ type EndpointSpec struct {
 	NATDiscoveryPort *int32            `json:"natDiscoveryPort,omitempty"`
 }
 
+const (
+	UDPPortBackendConfig               = "udp-port"
+	ForceUDPEncapsulationBackendConfig = "force-udp-encaps"
+)
+
+// This list should contain only keys that configure node specific settings via labels
```

Comment:
@tpantelis following your comments I've made it clear here that we should think twice before adding other valid parameters, for example selection of the cable driver, or anything that would be cluster or inter-cluster -wide.